google_health/delphi_google_health/run.py

```diff
@@ -27,19 +26,41 @@
                         PULL_START_DATE)
 
 
-def run_module():
-    """Main function run when calling the module.
+def run_module(params):
+    """
+    Main function run when calling the module.
 
     Inputs parameters from the file 'params.json' and produces output data in
     the directory defined by the `export_dir` (should be "receiving" expect for
     testing purposes).
+
+    Parameters
+    ----------
+    params
+        Dictionary containing indicator configuration. Expected to have the following structure:
+        - "common":
+            - "export_dir": str, directory to write output
+            - "log_exceptions" (optional): bool, whether to log exceptions to file
+            - "log_filename" (optional): str, name of file to write logs
+        - "indicator":
+            - "static_file_dir": str, path to DMA mapping files
+            - "data_dir": str, location of cached CSVs
+            - "start_date": str, YYYY-MM-DD format, first day to generate data for
+            - "end_date": str, YYYY-MM-DD format or empty string, last day to generate data for.
+            - "ght_key": str, GHT API key
+            - "wip_signal": list of str or bool, to be passed to delphi_utils.add_prefix
+            - "test": str, whether to run in test mode
```

Comment:
I think we can remove this parameter entirely and just use the existence of `test_data_dir` to tell if we are in test mode (make sure to mark it as optional).

Reply:
:+1: